File: NpvCalculator.Core/Classes/NetPresentValueResponse.cs
using System.Collections.Generic;

namespace NpvCalculator.Core.Classes
{
    public class NetPresentValueResponse
    {
        public string Name { get; set; }
        public double InitialInvestment { get; set; }
        public double LowerBoundDiscountRate { get; set; }
        public double UpperBoundDiscountRate { get; set; }
        public double DiscountRateIncrement { get; set; }
        public IEnumerable<PeriodAmount> CashFlows { get; set; }
        public IEnumerable<NetPresentValuePerRate> Results { get; set; }
    }
}

File: NpvCalculator-Angular/src/app/_models/net-present-value-response.ts
import { NetPresentValuePerRate } from './net-present-value-per-rate';
import { PeriodAmount } from './period-amount';

export interface NetPresentValueResponse {
    netPresentValueId: number;
    name: string;
    initialInvestment: number;
    lowerBoundDiscountRate: number;
    upperBoundDiscountRate: number;
    discountRateIncrement: number;
    cashFlows: PeriodAmount[];
    results: NetPresentValuePerRate[];
}
